elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -61,7 +61,8 @@ def __init__(
         self.combiner = combiner
         self.embedding_service_endpoint = embedding_service_endpoint
         self.tape = None
-        self.lookup_func = None
+        # function args: (layer name, embedding id list)
```

Comment:
Move this comment to some docstring or to line 136 instead?

Reply:
Done. Move this comment to a docstring.